README.md

```diff
@@ -1,2 +1,30 @@
 # Restaurant
 This is a restaurant application built with Javascript ES6 and Webpack
+## Built with
+
+- Pure JavaScript
+- DOM properties and attributes
+- HTML & CSS
+
+## Getting Started
+To get a local copy of the repository please run the following commands on your terminal:
+```$ cd <folder> ```
+```$ git clone https://github.com/MahmoudBakr23/Restaurant.git ```
+```cd Restaurant ```
+
```

Comment:
### Add a live link to the page.
- [ ]  In order to make sure we have a more descriptive README.mdm it would be great if we added a live link to where the page is hosted live and more instructions of how to run the project locally (maybe using `web-dev-server`)([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))